src/trials.jl

```diff
@@ -455,7 +455,8 @@ function Base.show(io::IO, ::MIME"text/plain", t::Trial)
     end
 
     print(io, "\n")
-    for histrow in eachrow(hist)
+    for r in axes(hist, 1)
+        histrow = view(hist, r, :)
         print(io, "\n", pad, "  ")
         for (i, bar) in enumerate(histrow)
```

Comment:
Why not replace `enumerate(histrow)` with `enumerate(hist[r,:])` ?

Reply:
Sure, I've changed it to skip the `histrow` assignment. (`enumerate(view(hist, r, :))` to avoid the allocation with array slicing.)

Reply:
Cool.